examples/contrib/cifar10/main.py

```diff
@@ -203,13 +203,13 @@ def run(
 
 def get_dataflow(config):
     # - Get train/test datasets
-    if idist.get_rank() > 0:
+    if idist.get_local_rank() > 0:
         # Ensure that only rank 0 download the dataset
```

Comment:
```suggestion
        # Ensure that only local rank 0 download the dataset
        # Thus each node will download a copy of the dataset
```

@Moh-Yakoub please update the comment here and below. Thanks

Reply:
@Moh-Yakoub below as well please